src/connectors/aws_collect.py

```diff
@@ -1372,6 +1393,26 @@
             ],
         },
     },
+    'ec2.describe_snapshots':{
+      'response': {
+            'Snapshot': {
```

Comment:
because the `Snapshots` returned by AWS is a list, you need to use a list `[...]` instead of a dict `{...}` here.